ephios/settings.py

```diff
@@ -47,7 +47,9 @@
     SECURE_SSL_REDIRECT = True
     SECURE_HSTS_SECONDS = 3600
```

Comment:
from the django docs: "Once you confirm that all assets are served securely on your site (i.e. HSTS didn’t break anything), it’s a good idea to increase this value so that infrequent visitors will be protected (31536000 seconds, i.e. 1 year, is common)."

HSTS preloading set below will also only work if this value is set so smth >1 year (compare https://hstspreload.org/)

Reply:
Ah thanks for point me to that site.